lib/platform/azure/types.ts

```diff
@@ -0,0 +1,5 @@
+import { Pr } from '../common';
+
+export interface AzurePr extends Pr {
+  sourceRefName?: string;
```

Comment:
I think this might be the same as `branchName` in `Pr`.

Reply:
I was unsure. can you check that? If true, we can remove that here

Reply:
As far as I can tell from [the documentation](https://docs.microsoft.com/en-us/rest/api/azure/devops/git/pull%20requests/get%20pull%20request?view=azure-devops-rest-6.0#gitpullrequest):
> sourceRefName	The name of the source branch of the pull request.
> targetRefName The name of the target branch of the pull request.